### ISSUE TITLE
KITTI Training Set Detections

### ISSUE THREAD

**Issue by Paulkie99** [noahcao/OC_SORT#126]
Hello, thank you very much for making detections on the KITTI test set publicly available.

Would it be possible to do the same for the KITTI training set?

Thank you very much!
